sympy/ntheory/tests/test_factor_.py

```diff
@@ -522,7 +522,7 @@ def test_factorrat():
     assert str(factorrat(Rational(1, 1), visual=True)) == '1'
     assert str(factorrat(S(25)/14, visual=True)) == '5**2/(2*7)'
     assert str(factorrat(Rational(25, 14), visual=True)) == '5**2/(2*7)'
-    assert str(factorrat(S(-25)/14/9, visual=True)) == '-5**2/(2*3**2*7)'
+    assert str(factorrat(S(-25)/14/9, visual=True)) == '(-1)*5**2/(2*3**2*7)'
```

Comment:
We should get rid of these parentheses around the `-1` and check:
```python
>>> S('-3*4',evaluate=False)
(-3)*4
>>> Mul(-3,4,evaluate=False).args
(-3, 4)
>>> S('-3*4',evaluate=False).args
(4, -3)
>>> S('3*4',evaluate=False).args
(3, 4)
```

Reply:
This is controlled be precedence of a negated factor in a product while gets precedence of Add and thus is parenthesized. This is a separate issue.